kafka-protocol-binding.md

```diff
@@ -246,12 +246,16 @@ content-type: application/cloudevents+json; charset=UTF-8
 
 {
     "specversion" : "1.0-rc1",
-    "datacontenttype" : "com.example.someevent",
+    "type" : "com.example.someevent",
+	"source" : "/mycontext/subcontext",
+	"id" : "1234-1234-1234",
+	"time" : "2018-04-05T03:56:24Z",
+    "datacontenttype" : "application/json",
```

Comment:
Perhaps use application/xml to make apparent that the data content type can differ from the content type of the message itself (similar to how it's shown in the JSON event format spec)?

Reply:
Done. What's funny about you focusing on this is that I was tempted to add a sentence in there about how datacontenttype is optional in this case since the data payload is encoded the same way as the envelope - but decided not to at the last minute.  Using "XML" doesn't give me that option :-)

Reply:
Yes, I was thinking the same :) Either get rid of it or make actually use of it to define a different content type for data.